content/md/productivity/manage-your-rules.md

```diff
@@ -102,8 +103,10 @@ Depending on the action you choose, the template changes. The table below lists
 | Clear categories  | Delete all the categories of your product/product model selection  |
 | Copy attribute value  | Copy the value from an attribute to another one  |
 | Remove categories  | Remove the categories of your product/product model selection  |
-| Set attribute value  | Set value to a multi select, number or boolean attribute. It will remove the former values |
+| Add attribute value  | Add values to a multi select or price collection. It will keep the former values |
+| Set attribute value  | Set values to a multi select, simple select, number, text, price collection or boolean attribute. It will remove the former values |
```

Comment:
same comment as the previous one. I don't see "price collection" as an attribute type.

Reply:
The "price" type is a collection of prices in multiple currencies (one price per actrivated currency). That's why we talk about an attribute of "price collection" type. I see "Price collection" in other pages in the helpcenter so I guess it's ok.